acceptance/openstack/placement/v1/resourceproviders_test.go

```diff
@@ -0,0 +1,25 @@
+package v1
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+	"github.com/gophercloud/gophercloud/openstack/placement/v1/resourceproviders"
+	th "github.com/gophercloud/gophercloud/testhelper"
+)
+
+func TestResourceProviderList(t *testing.T) {
```

Comment:
I'm getting a 403 error when I try to run this test as a normal user -- I think this requires admin access?

If so, then we need to add:

```go
clients.RequireAdmin(t)
```

at the very beginning of this test function.

I'll make this change right after I merge this PR.